lib/vacations.ex

```diff
@@ -0,0 +1,23 @@
+defmodule GlassFactoryApi.Vacations do
+  @moduledoc """
+
+  """
+
+  alias GlassFactoryApi.ApiClient
+  alias GlassFactoryApi.Vacations.Vacation
+
+  @doc """
+
+  """
```

Comment:
I think you forgot to remove this

Reply:
I will add documentation, how I would like feedback from you, I've pushed without all things the I would like to do in this PR. I've added a WIP flag :)